alws/models.py

```diff
@@ -733,6 +733,8 @@ class User(SQLAlchemyBaseUserTable[int], Base):
     last_name = sqlalchemy.Column(sqlalchemy.String(320), nullable=True)
     hashed_password: str = sqlalchemy.Column(
         sqlalchemy.String(length=1024), nullable=True)
+    is_active: bool = sqlalchemy.Column(
```

Comment:
not sure what we really need here type hint

Reply:
copy-paste :)